modules/standard/Random.chpl

```diff
@@ -123,6 +262,20 @@ class RandomStream {
       RandomStreamPrivate_lock$;
   }
 
+  /*
+    Skip to an arbitrary value in the random stream and return it.
+    This is equivalent to :chpl:proc:`skipToNth()` followed by
+    :chpl:proc:`getNext()`.
+
+    :arg n: The position in the stream to skip to.  Must be non-negative.
+    :type n: integral
+
+    :arg parSafe: Permits :chpl:param:`RandomStream.parSafe` to be overridden for this call.  Defaults to this.parSafe.
+    :type parSafe: bool
+
+    :returns: The `n`th value in the random stream as a `real(64)`.
+  */
+
```

Comment:
It looks like you added the return type to getNext() but not to getNth() here.  Is one strategy preferred?